README.md

```diff
@@ -47,34 +47,19 @@ environment for these tests, then you should activate it, otherwise you should
 create a new one. For more information on working with virtual environments see
 our [summary][virtualenv].
 
-### Install dependencies
```

Comment:
Yes I agree that `Tox` is now creating virtual envs. 

But maybe for the good practice and clarity we should always create new virtual env? 
Because installing tox (`pip install tox`) we also install packages like `pkg-resources`, `pluggy`, `py`, `tox` and `virtualenv`. And it will just add more "noise" into our pure python.

What do you think?